dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -169,8 +170,16 @@ internal open class RumViewScope(
         if (stopped) return
 
         if (activeActionScope != null) {
-            devLogger.w(ACTION_DROPPED_WARNING.format(Locale.US, event.type, event.name))
-            return
+            if (event.type == RumActionType.CUSTOM && !event.waitForStop) {
```

Comment:
Maybe you could extract this in a separated method for visibility

Reply:
cannot come up with a good naming tbh, so I think I will leave it like this